docs-ref-services/servicebus.md

```diff
@@ -17,131 +17,77 @@ ms.service: service-bus
-    // 'Shared Access policies' section.
-    private static final String connectionString = "{connection string}";
-    private static final String queueName = "{queue name}";
-    private static IQueueClient queueClient;
-    private static int totalSend = 100;
-    private static int totalReceived = 0;
-
-    public static void main(String[] args) throws Exception {
-
-        Log.log("Starting BasicSendReceiveWithQueueClient sample");
-
-        // create client
-        Log.log("Create queue client.");
-        queueClient = new QueueClient(new ConnectionStringBuilder(connectionString, queueName), ReceiveMode.PeekLock);
-
-        // send and receive
-        queueClient.registerMessageHandler(new MessageHandler(queueClient), new MessageHandlerOptions(1, false, Duration.ofMinutes(1)));
-        for (int i = 0; i < totalSend; i++) {
-            int j = i;
-            Log.log("Sending message #%d.", j);
-            queueClient.sendAsync(new Message("" + i)).thenRunAsync(() -> { Log.log("Sent message #%d.", j);});
-        }
-
-        while(totalReceived != totalSend) {
-            Thread.sleep(1000);
-        }
-
-        Log.log("Received all messages, exiting the sample.");
-        Log.log("Closing queue client.");
-        queueClient.close();
-    }
-
-    static class MessageHandler implements IMessageHandler {
-        private IQueueClient client;
-
-        public MessageHandler(IQueueClient client) {
-            this.client = client;
-        }
-
-        @Override
-        public CompletableFuture<Void> onMessageAsync(IMessage iMessage) {
-            Log.log("Received message with sq#: %d and lock token: %s.", iMessage.getSequenceNumber(), iMessage.getLockToken());
-            return this.client.completeAsync(iMessage.getLockToken()).thenRunAsync(() -> {
-                Log.log("Completed message sq#: %d and locktoken: %s", iMessage.getSequenceNumber(), iMessage.getLockToken());
-                totalReceived++;
-            });
-        }
-
-        @Override
-        public void notifyException(Throwable throwable, ExceptionPhase exceptionPhase) {
-            Log.log(exceptionPhase + "-" + throwable.getMessage());
-        }
-    }
-}
-```
-
 > [!div class="nextstepaction"]
-> [Explore the Client APIs](/java/api/overview/azure/servicebus/client)
-> [Find more examples here (See also above for more details)](https://github.com/Azure/azure-service-bus/blob/master/samples/Java/)
+> [Explore the Management APIs](/java/api/overview/azure/servicebus/management)
 
-## Management API
+## Libraries for data access
 
-Create and manage namespaces, topics, queues, and subscriptions with the management API.
+The official Service Bus client is available in [source code form on GitHub](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/servicebus/) and
+binaries and packaged sources [are available on Maven Central](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22azure-messaging-servicebus%22).
```

Comment:
The above information regarding maven is already in the table below. We can add a column for the Github link and remove the above.

Instead, we should have something here that talks about what the data access libraries would give you like we did with the one line description for the management plane SDK a few lines above.

You can use something like this:

> Use the data access libraries to send and receive messages from Service Bus queues, topics and subscriptions. There are two libraries available that provide these features at the moment

Then have a section (h3) for the Track 2 library, give links, the pom xml. Then another section (h3) for Track 1 with similar info